### DISCUSSION TITLE
求助v3.30更新后无法正常启动，系统是windows7 64位

### DISCUSSION THREAD

**Discussion by Huyang-boy** [AlistGo/alist#5831] [Q&A] [answered]
v3.29.1在系统上正常启动
![Screenshot_2024-01-06-09-07-14-171_com.oray.sunlogin.jpg](https://github.com/alist-org/alist/assets/79417371/890f937a-2edd-477b-9c78-015c1ed0612a)
![Screenshot_2024-01-06-09-08-29-336_com.oray.sunlogin.jpg](https://github.com/alist-org/alist/assets/79417371/a7b74027-3721-4883-a93e-55e9b65b4306)

v3.30无法启动
![Screenshot_2024-01-06-08-23-40-061_com.oray.sunlogin.jpg](https://github.com/alist-org/alist/assets/79417371/49994a30-26ea-4a2d-a573-f7b0eca69152)

**Comment by Huyang-boy** [comment]
应该是windows7已经不支持3.29以后的版本了

**Comment by anwen-anyi** [comment, marked as answer]
GO 1.21.+ 高版本  已经不支持 Windows7

- https://github.com/golang/go/issues/64622

**Comment by NERO-Tsui** [comment]
o(╥﹏╥)o win版都试了一遍，就知道可能是go寄了..

**Comment by pongfcnkl** [comment]
自己手动用1.21.4版本编译或者还是尽快升级到win10吧
